arviz/plots/backends/bokeh/posteriorplot.py

```diff
@@ -187,6 +187,15 @@ def display_rope(max_data):
             line_color="red",
             line_alpha=0.7,
         )
+        less_than_left_probability = (values < vals[0]).mean()
+        less_than_right_probability = (values <= vals[1]).mean()
+        probability_within_rope = less_than_right_probability - less_than_left_probability
```

Comment:
```suggestion
        probability_within_rope = ((values > vals[0]) & (values <= vals[1])).mean()
```

I think this will perform better in terms of memory usage, values are not generally too large arrays but I think it's better to be cautious. It should return the same value.